docs/Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN \
   apk add --no-cache \
     git \
     git-fast-import \
+    curl \
+    jq \
```

Comment:
I don't know when this Dockerfile is used, but if it is used as runner for the publish workflow it is easier to pur k0s binary download process here